scripts/ai2_internal/run_with_beaker.py

```diff
@@ -36,7 +36,8 @@ def main(param_file: str, args: argparse.Namespace):
     params = Params.from_file(param_file, overrides, ext_vars)
 
     # Write params as json. Otherwise Jsonnet's import feature breaks.
-    compiled_params_path = tempfile.mkstemp(".json", "config")[1]
+    params_dir = tempfile.mkdtemp(prefix="config")
```

Comment:
This does leak this directory but the script seems to leak temporary files generally, I assume this is to allow users look at the config for debugging etc. Please let me know if we would like to clean up the directory at the end instead.

Reply:
Yes, this is handy for debugging. Should be fine as-is. Someone would have to create an insane amount of beaker jobs before the periodic cleaning of the temp directory wouldn't be enough.